package.json

```diff
@@ -11,15 +11,7 @@
     "type": "git",
     "url": "http://github.com/graphql/graphql-relay-js.git"
   },
-  "main": "lib/index.js",
-  "directories": {
```

Comment:
`directories` and `files` are no longer needed since we'll be publishing directly from the `dist` directory